app-crypt/zulucrypt/zulucrypt-5.4.0_pre20180223.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils vcs-snapshot
+
+DESCRIPTION="Front end to cryptsetup"
+HOMEPAGE="https://mhogomchungu.github.io/zuluCrypt/"
+MY_COMMIT="76637bb05af13744bf1734b56f67d6d5cc2343b1"
+SRC_URI="https://github.com/mhogomchungu/zuluCrypt/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gnome kwallet +qt5 udev"
+
+REQUIRED_USE="kwallet? ( qt5 )"
+
+CDEPEND="
```

Comment:
Please don't use single-letter+`DEPEND` variables, those names are reserved for EAPI use.